src/map.cpp

```diff
@@ -2287,8 +2025,7 @@ bool ServerMap::saveBlock(MapBlock *block, MapDatabase *db)
 	o.write((char*) &version, 1);
 	block->serialize(o, version, true);
 
-	std::string data = o.str();
-	bool ret = db->saveBlock(p3d, data);
+	bool ret = db->saveBlock(p3d, o.str());
```

Comment:
i embed this tiny string copy removal (compiler should do it in release but it's there in debug and it has a little overhead)